src/main/java/teammates/ui/controller/InstructorCourseEnrollPageAction.java

```diff
@@ -21,10 +22,10 @@ public ActionResult execute() {
         new GateKeeper().verifyAccessible(
                 instructor, logic.getCourse(courseId), Const.ParamsNames.INSTRUCTOR_PERMISSION_MODIFY_STUDENT);
         
+        courseId = Sanitizer.sanitizeForHtml(courseId);
+        studentsInfo = Sanitizer.sanitizeForHtml(studentsInfo);
         /* Setup page data for 'Enroll' page of a course */
-        InstructorCourseEnrollPageData pageData = new InstructorCourseEnrollPageData(account);
-        pageData.courseId = courseId;
-        pageData.enrollStudents = studentsInfo;
+        InstructorCourseEnrollPageData pageData = new InstructorCourseEnrollPageData(account, courseId, studentsInfo);
```

Comment:
It's probably best to sanitise on the `*PageData`.
We will probably migrate it all to `c:out` eventually, but it's best to keep them purely on the `*PageData` side for now.